### PR TITLE
Simplify blur

### PR DESCRIPTION
**Pull request by AashishDhakal**

### CODE REVIEW

File: video-editor/docker-compose.yml [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+      context: .
+      dockerfile: Dockerfile
+    container_name: video_editor_container
+    depends_on:
+      - alpr
+      - skew_correction
+    ports:
+      - "8081:8081"
+    environment:
+      - SDK_URL=http://alpr:8080
```
Comment (danleyb2) [inline video-editor/docker-compose.yml:38]: video-editor does not need **SDK_URL**

**Review by danleyb2 — CHANGES_REQUESTED** [review]

**Comment by AashishDhakal** [comment]
@marcbelmont @danleyb2 By default the docker compose appends prefix to the volume. For example, if not used external:true, the volume will be created as video-editor_license, and causes issues for the machines which has Snapshot SDK installed previously.

So, here are the steps:

If the machine has Snapshot SDK installed previously:
`docker-compose up --build`

Else:
`docker volume create license`
`docker-compose up --build`

**Comment by danleyb2** [comment]
@AashishDhakal I prefer making the default option to be storing the license in an external volume so it's easier for a user to re-use existing licenses

@marcbelmont LGTM

**Review by danleyb2 — APPROVED** [review]

**Comment by marcbelmont** [comment]
LGTM

**Comment by AashishDhakal** [comment]
@marcbelmont I think I don't have access to the docs repo.

**Comment by marcbelmont** [comment]
@AashishDhakal 

I just added you

**Pull request merged**